common/src/main/java/com/wynntils/mc/render/RenderUtils.java

```diff
@@ -337,33 +338,46 @@ public static void drawTexturedRectWithColor(
         RenderSystem.disableBlend();
     }
 
-    public static void drawArc(CustomColor color, int x, int y, int z, float fill, int radius) {
-        drawArc(new PoseStack(), color, x, y, z, fill, radius);
+    public static void drawArc(CustomColor color, int x, int y, int z, float fill, int innerRadius, int outerRadius) {
+        drawArc(new PoseStack(), color, x, y, z, fill, innerRadius, outerRadius);
     }
 
-    public static void drawArc(PoseStack poseStack, CustomColor color, int x, int y, int z, float fill, int radius) {
+    public static void drawArc(
+            PoseStack poseStack, CustomColor color, int x, int y, int z, float fill, int innerRadius, int outerRadius) {
         // keeps arc from overlapping itself
         int segments = (int) Math.min(fill * MAX_CIRCLE_STEPS, MAX_CIRCLE_STEPS - 1);
```

Comment:
Can't we increase MAX_CIRCLE_STEPS now?

Reply:
It's the same step count as Wynntils so 🤷

Reply:
Ah, it should be fine then.